File: 프로그래머스/3/12938. 최고의 집합/최고의 집합.js
function solution(n, s) {
    if (s < n){
        return [-1]
    }
    
    const q = parseInt(s/n)
    const r = parseInt(s%n)
    
    const answer = []
    
    for (let i=0; i< n-r; i++){
        answer.push(q)
    }
    
    for(let i=0; i < r; i++){
        answer.push(q+1)
    }
    
    return answer
    
}

File: 프로그래머스/3/12938. 최고의 집합/최고의 집합.py
def solution(n, s):
    if s < n:
        return [-1]
    q = s // n  
    r = s % n  
    answer = []

    for _ in range(n - r):
        answer.append(q)

    for _ in range(r):
        answer.append(q + 1)

    return answer
